tests/test_crawlers.py

```diff
@@ -390,6 +415,30 @@ def test_intersects_time_range_no_upper_limit(self):
         # no upper limit and no start_time, without intersection
         self.assertFalse(crawler._intersects_time_range(None, datetime(2019, 2, 19)))
 
+    @mock.patch("geospaas_harvesting.crawlers.OpenDAPCrawler._http_get")
+    @mock.patch("geospaas_harvesting.crawlers.OpenDAPCrawler._get_links")
+    def test_correct_navigation_to_download_page_for_OSISAF(self, mock_get_link, mock_http_get):
+        """Test the functionality of "get_download_url" method for OpenDAP crawler of OSISAF project """
+        mock_get_link.return_value = [
+            '/thredds/dodsC/osisaf/met.no/ice/amsr2_conc/2019/11/ice_conc_nh_polstere-100_amsr2_201911301200.nc.html']
+        expected_urls = ['https://thredds.met.no/thredds/catalog/osisaf/met.no/ice/amsr2_conc/2019/11/catalog.html?dataset=osisaf/met.no/ice/amsr2_conc/2019/11/ice_conc_nh_polstere-100_amsr2_201911301200.nc', ]
+        request_link = crawlers.ThreddsCrawler.get_download_url(
+            crawlers.OpenDAPCrawler, expected_urls)
+        self.assertEqual(
+            request_link, 'https://thredds.met.no/thredds/dodsC/osisaf/met.no/ice/amsr2_conc/2019/11/ice_conc_nh_polstere-100_amsr2_201911301200.nc.dods')
+
+    @mock.patch("geospaas_harvesting.crawlers.OpenDAPCrawler._http_get")
+    @mock.patch("geospaas_harvesting.crawlers.OpenDAPCrawler._get_links")
+    def test_correct_navigation_to_download_page_for_OSISAF_with_none(self, mock_get_link, mock_http_get):
+        """Test the functionality of "get_download_url" method for OpenDAP crawler of OSISAF
+        project to return None in the case of incorrect link (lack of dodsC in link)"""
+        mock_get_link.return_value = [
+            '/thredds/osisaf/met.no/ice/amsr2_conc/2019/11/ice_conc_nh_polstere-100_amsr2_201911301200.nc.html']
+        expected_urls = ['testurl', ]
```

Comment:
Same comment as https://github.com/nansencenter/django-geo-spaas-harvesting/pull/20/files#r472883483
Also, if you are not going to put an example of valid value, you might as well use an empty string because it's never used

Reply:
Done in the next commit